content/docs/started/kubeflow-overview.md

```diff
@@ -0,0 +1,160 @@
+in the "Doc diagrams" folder in the public Kubeflow shared drive.
+-->
+
+This guide introduces Kubeflow as a platform for developing and deploying a
+machine learning (ML) system.
+
+Kubeflow is a platform for data scientists who want to build and experiment with
+ML pipelines. Kubeflow is also for ML engineers and operational teams who want
+to deploy ML systems to various environments for development, testing, and
+production-level serving.
+
+## Architectural overview
+
+Kubeflow is *the ML toolkit for Kubernetes*.
+The following diagram shows Kubeflow as a platform for arranging the
+components of your ML system on top of Kubernetes:
+
+<img src="/docs/images/kubeflow-overview-platform-diagram.svg" 
+  alt="An architectural overview of Kubeflow on Kubernetes"
+  class="mt-3 mb-3 border border-info rounded">
+
+Kubeflow builds on [Kubernetes](https://kubernetes.io/) as a system for 
+deploying, scaling, and managing complex systems.
+
+Using the Kubeflow configuration interfaces (see [below](#interfaces)) you can
+specify the ML tools required for your workflow. Then you can deploy the 
+workflow to various clouds, local, and on-prem platforms for experimentation and 
+for production use.
+
+## Introducing the ML workflow
+
+When you develop and deploy an ML system, the ML workflow typically consists of 
+a number of stages. Developing an ML system is an iterative process. 
+You need to evaluate the output of various stages of the ML workflow, and apply
+changes to the model and parameters when necessary to ensure the model keeps 
+producing the results you need.
+
+For the sake of simplicity, the following diagram
+shows the workflow stages in sequence, with just one arrow at the end pointing
+back into the flow as an indication of the iterative nature of the process:
+
+<img src="/docs/images/kubeflow-overview-workflow-diagram-1.svg" 
+  alt="A typical machine learning workflow"
+  class="mt-3 mb-3 border border-info rounded">
+
+Looking at the stages in more detail:
+
+* In the experimental phase, you develop your model based on initial
+  assumptions, and test and update the model iteratively to produce the
+  results you're looking for:
+
+  * Identify the problem you want the ML system to solve.
+  * Collect and analyse the data you need to train your ML model.
+  * Choose an ML framework and algorithm, and code the initial version of your 
+    model.
+  * Experiment with the data and with training your model.
+  * Tune the model hyperparameters to ensure the most efficient processing and the
+    most accurate results possible.
+
+* In the production phase, you deploy a system that performs the following 
+  processes:
+
+  * Transform the data into the format that your training system needs. The
+    transformation process must be the same in the experimental and production
+    phases, to ensure that the ML model bases its predictions on the same data 
+    format as that used during training.
```

Comment:
Hmm, the words "as the model used during training" are ambiguous - they could refer to the model rather than the data used during training. I'll leave the wording as is, because putting "data" back in there is too much data. :)